File: Java/Account.java
class Account {
        Integer id;
        String name;
        String document;
        String email;
        String password;

public Account(String name, String document){
        this.name = name;
        this.document = document;
}
/*Aqui agragamos el metodo public para que el metodo car pueda alimentar a su parametro drive
que a su vez se alimenta de la super clase Account que esta designada como su tipo de dato*/
}

File: Python/account.py
class Account:
    id = int
    name = str
    document = str
    email = str
    password = str

    def __init__(self, name, document):
        self.name = name
        self.document = document

# Como ya vimos en clases pasadas debemos poner el atributo self antes de poner los atributos en este caso name y document
